Tests/designspaceLib/data/test_avar2.designspace

```diff
@@ -0,0 +1,1481 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="5.0">
+  <axes>
+    <axis tag="jstf" name="Justify" minimum="-100" maximum="100" default="0"/>
+    <axis tag="wght" name="Weight" minimum="100" maximum="900" default="400">
+      <map input="100" output="26"/>
+      <map input="200" output="39"/>
+      <map input="300" output="58"/>
+      <map input="400" output="90"/>
+      <map input="500" output="108"/>
+      <map input="600" output="128"/>
+      <map input="700" output="151"/>
+      <map input="800" output="169"/>
+      <map input="900" output="190"/>
+    </axis>
+    <axis tag="wdth" name="Width" minimum="62.5" maximum="100" default="100">
+      <map input="62.5" output="70"/>
+      <map input="75" output="79"/>
+      <map input="87.5" output="89"/>
+      <map input="100" output="100"/>
+    </axis>
+    <mappings>
+      <mapping>
+        <input>
+	  <dimension name="Justify" xvalue="0"/>
+        </input>
+        <output>
+	  <dimension name="Weight" xvalue="90"/>
+        </output>
+      </mapping>
+      <mapping>
+        <input>
+	  <dimension name="Justify" xvalue="-100"/>
+        </input>
+        <output>
+	  <dimension name="Weight" xvalue="26"/>
+        </output>
+      </mapping>
+      <mapping>
+        <input>
+	  <dimension name="Justify" xvalue="+100"/>
```

Comment:
the "+" symbol is more like an embellishment here, I happens to work because we eval this as python float but I would rather stick to plain digits